pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.cs

```diff
@@ -738,8 +785,22 @@ public void UpdateUI(bool selectionChanged = true)
                         }
                         adducts.AddRange(showAdducts.Where(a => charges.Contains(Math.Abs(a.AdductCharge)) && !adducts.Contains(a))); // And the unranked charges as well
 
-                        var spectrumInfo = _selectedLibrary.GetSpectra(_peptides[index].Key, null, LibraryRedundancy.best).FirstOrDefault();
-                        var spectrumInfoR = LibraryRankedSpectrumInfo.NewLibraryRankedSpectrumInfo(spectrum,
+                        //Get all redundant spectrum for the selected peptide, add them to comboRedundantSpectra

```

Comment:
Totally nitpicking here, but our commenting style has a leading space (and, good, capital letter)

Reply:
(But, a wise coder once told me that "The only proper code formatting style is the one that's already in use." The idea is not to trip up the eye of whoever is digging through your code looking for who-knows-what.)